Add cheat sheet for Conan 2.0 (#3229)
conan-io/docs: knowledge/cheatsheet.rst

***********
Cheat sheet
***********

> JFrog has created the following visual cheat sheet for basic Conan commands and
> concepts which users can print out and use as a handy reference. It is available

Reviewer: ```suggestion
This is a visual cheat sheet for basic Conan commands and
concepts which users can print out and use as a handy reference. It is available
```
Everything in these docs is created by JFrog, this reads a bit weird
